Engine/src/Engine/Scene.h

```diff
@@ -76,11 +77,14 @@ namespace Engine
 
 		//Create quad for sprites
         VerticesComponent createSprite();
+        VerticesComponent createRectangle();
```

Comment:
I'm still confused why we should create a Rectangle separately from a sprite. I understand wanting to create different geometric objects, but in this case the rectangle is practically identical to a sprite, just without UV coords (which aren't used anyway if a texture isn't bound). Is there a reason for this? 

Additionally, these additional object creations should wait until we create a proper renderer class to store 'renderables'

Reply:
Check our convo on discord general for this lol. Basically GL stride offset of its vbos gets set based on 5 values with sprites but only three for rectangles. In other words, the gl binded data is slower, especially considering all the tiles don't need to be bounded to a texture when they're flat colored or with a gradient. May as well make it efficient since the texture will be bounded for each instance otherwise

Reply:
Based on our convo in person, this would be based on shaders, right? Could we alter the shader file we use for this? Worth discussing in the rendering refactor